sympy/printing/llvmjitcode.py

```diff
@@ -0,0 +1,175 @@
+from __future__ import print_function, division
+
+'''
+Use llvmlite to create executable functions from Sympy expressions
+
+  Prerequisites
+  -------------
+  LLVM - http://llvm.org/
+  llvmlite - https://github.com/numba/llvmlite
+
+Examples:
+
+Scalar expression
+-----------------
+  import sympy.printing.llvmlitecode as g
+
+  a = Symbol('a')
+  e = a*a + a + 1
+  e1 = g.get_jit_callable(e, args=[a])
+  print(e1(1.1), e.subs('a',1.1))
+'''
+
+import sympy
+import ctypes
+
+from sympy.external import import_module
+
+ll = import_module('llvmlite.ir').ir
+llvm = import_module('llvmlite.binding').binding
+
+llvm.initialize()
+llvm.initialize_native_target()
+llvm.initialize_native_asmprinter()
+
+
+class LLVMJitPrinter(sympy.printing.printer.Printer):
+    def __init__(self, *args, **kwargs):
```

Comment:
This class and `__init__` method needs a docstring. Note that we follow the numpydoc style.